### PR TITLE
Ch2 minor

### PR DESCRIPTION
**Pull request by csgillespie**

### CODE REVIEW

File: 02-set-up.Rmd [hunk 1]
```diff
@@ -352,7 +352,7 @@ R --vanilla
 
 ### An overview of R's startup files
 
-Two script files run each time R starts (unless one of the command line options outlined above is used):
+Two files are read each time R starts (unless one of the command line options outlined above is used):
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: Good point

File: 02-set-up.Rmd [hunk 2]
```diff
@@ -857,7 +857,8 @@ Most of these are self-explanatory but it is worth mentioning a few that can boo
 - R version settings allow RStudio to 'point' to different R versions/interpreters, which may be faster for some projects.
 
 - `Restore .RData`: Unticking this default preventing loading previously creating R objects. This will make
-starting R quicker and also reduce the change of getting bugs due to previously created objects.
+starting R quicker and also reduce the change of getting bugs due to previously created objects. For this
```
Comment (Robinlovelace) [inline 02-set-up.Rmd]: Should be **chance** of getting bugs.

Comment (csgillespie) [inline 02-set-up.Rmd]: Fixed

**Comment by Robinlovelace** [comment]
Seems the build is failing:

https://travis-ci.org/csgillespie/efficientR/builds/162790862

**Pull request merged**